scripts/configure_dev-deps.sh

```diff
@@ -14,3 +14,23 @@ function install_go_module {
 install_go_module golang.org/x/lint/golint
 install_go_module golang.org/x/tools/cmd/stringer
 install_go_module github.com/go-swagger/go-swagger/cmd/swagger
+
+OS=$(uname)
+
+if [ "$OS" == "Linux" ]
+then
+    # This script is called from multiple locations, so to be safe we
+    # will do this check and not assume it's already been done.
+    if ! which sudo > /dev/null
+    then
+        apt-get update
+        apt-get install sudo -y
+    fi
+
+    sudo apt-get update
+    sudo apt-get install shellcheck -y
+elif [ "$OS" == "Darwin" ]
+then
+    brew install shellcheck
+fi
```

Comment:
All that logic need to go to `configure-dev.sh`

Reply:
I agree that is where it should go, but I don't think I can move that there without some extra work first.  Here is a grep that shows where else that file is called:

```
~/projects/go-algorand:master$ ag -C 2 configure_dev-deps.sh
docker/build/Dockerfile-deploy
22-    NETWORK=${NETWORK}
23-WORKDIR $GOPATH/src/github.com/algorand/go-algorand
24:RUN scripts/configure_dev-deps.sh && make deps && make clean && find tmp && TMPDIR/deploy_linux_version_exec.sh
25-ENTRYPOINT ["/bin/bash"]
26-

docker/build/Dockerfile
18-    PKG_ROOT=${PKG_ROOT}
19-WORKDIR $GOPATH/src/github.com/algorand/go-algorand
20:RUN scripts/configure_dev-deps.sh && make deps && make clean && scripts/local_install.sh -n
21-ENTRYPOINT ["/bin/bash"]
22-

scripts/build_release_centos_docker.sh
23-REPO_DIR="$( cd "$(dirname "$0")" ; pwd -P )"/..
24-
25:${REPO_DIR}/scripts/configure_dev-deps.sh
26-
27-cd ${REPO_DIR}

scripts/configure_dev.sh
33-fi
34-
35:${SCRIPTPATH}/configure_dev-deps.sh
36-
```

Reply:
-> For the two docker files, you need to add the spellcheck to the `RUN apt` line.
-> for the build_release_centos_docker.sh, it need to be added to the `RUN yum install -y autoconf awscli git gnupg2 nfs-utils python36 sqlite3 boost-devel expect jq libtool gcc-c++ libstdc++-devel libstdc++-static rpmdevtools createrepo rpm-sign bzip2` line.
The last one is good and doesn't need to be modified.

Reply:
Sounds good.